canteen/orders/migrations/0001_initial.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.1.4 on 2018-12-20 13:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Dishes',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('dish_name', models.CharField(max_length=200)),
+                ('price', models.IntegerField()),
+                ('dish_logo', models.CharField(max_length=1000)),

```

Comment:
change charField to imagefield 
and define function for the location of image before putting it into db